scripts/setup/restore-backup

```diff
@@ -15,9 +15,15 @@ POSTGRES_PWENT = get_postgres_pwent()
 
 parser = argparse.ArgumentParser()
 parser.add_argument("tarball", help="Filename of input tarball")
+parser.add_argument(
+    "--skip-settings", help="Do not overwrite local /etc/zulip/settings.py", action="store_true"
+)
+parser.add_argument(
+    "--skip-zulipconf", help="Do not overwrite local /etc/zulip/zulip.conf", action="store_true"
```

Comment:
This doesn't match the documented `--keep-foo` naming scheme. I think I like `keep` better.

Reply:
Whoops, thanks for the catch.  Swapped to `keep`